### DISCUSSION TITLE
Latest requests done

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#393] [General]
[12-04-23.zip](https://github.com/elax46/custom-brand-icons/files/11210041/12-04-23.zip)

**Comment by rchiileea** [comment]
![image](https://user-images.githubusercontent.com/87599807/232206349-0ed64eed-8660-4f40-9852-ef9246a41096.png)
![image](https://user-images.githubusercontent.com/87599807/232206328-a7a9f4ba-2b81-4389-bb61-8425aa8d554a.png)
[15-04-23.zip](https://github.com/elax46/custom-brand-icons/files/11239015/15-04-23.zip)

**Comment by rchiileea** [comment]
[27042023.zip](https://github.com/elax46/custom-brand-icons/files/11350427/27042023.zip)

**Reply by elax46** [reply]
> [27042023.zip](https://github.com/elax46/custom-brand-icons/files/11350427/27042023.zip)

Hive icon doesn't have a single path I'm waiting for you to update it to release all the new icons added #399 @rchiileea

**Comment by rchiileea** [comment]
[tronity.zip](https://github.com/elax46/custom-brand-icons/files/11361858/tronity.zip)
[hive.zip](https://github.com/elax46/custom-brand-icons/files/11361866/hive.zip)
corrected tronity one (did not translate as well as i thought in ha and hive redone

**Comment by rchiileea** [comment]
[12052023.zip](https://github.com/elax46/custom-brand-icons/files/11461285/12052023.zip)

**Comment by rchiileea** [comment]
added latest icons, just replace hbo max one in repoo with the h
[210523.zip](https://github.com/elax46/custom-brand-icons/files/11524157/210523.zip)
bo one from this due to branding change by hbo.

**Reply by elax46** [reply]
> added latest icons, just replace hbo max one in repoo with the h [210523.zip](https://github.com/elax46/custom-brand-icons/files/11524157/210523.zip) bo one from this due to branding change by hbo.

Thanks for your amazing work, as always

**Comment by elax46** [comment]
Thank you for your contribution @raymercury  I remind you that as a collaborator you can approve the pull requests yourself then if you want me to double check I remind you to assign me the task because if I don't get a notification I will lose the request for days

**Comment by rchiileea** [comment]
finally back home from france that was a extended stay.

**Comment by rchiileea** [comment]
[lg.zip](https://github.com/elax46/custom-brand-icons/files/11934926/lg.zip)

**Comment by rchiileea** [comment]
[180723.zip](https://github.com/elax46/custom-brand-icons/files/12076958/180723.zip)
Latest updates @elax46

**Comment by rchiileea** [comment]
[01-08-23.zip](https://github.com/elax46/custom-brand-icons/files/12226268/01-08-23.zip)

**Comment by rchiileea** [comment]
[010923.zip](https://github.com/elax46/custom-brand-icons/files/12496841/010923.zip)

**Reply by elax46** [reply]
Thanks for your work, if you have time, you could make an icon for me too for sonoff's NSPanel Pro, thanks

**Reply by rchiileea** [reply]
done

**Comment by rchiileea** [comment]
> onoff's NSPanel Pro

![image](https://github.com/elax46/custom-brand-icons/assets/87599807/3fa6d1c0-2fb6-4af1-896c-9b6c7b44c127)
[nspro-panel.zip](https://github.com/elax46/custom-brand-icons/files/12511115/nspro-panel.zip)

**Comment by elax46** [comment]
> > onoff's NSPanel Pro
> 
> ![image](https://user-images.githubusercontent.com/87599807/265385079-3fa6d1c0-2fb6-4af1-896c-9b6c7b44c127.png) [nspro-panel.zip](https://github.com/elax46/custom-brand-icons/files/12511115/nspro-panel.zip)

Thanks so much

**Comment by rchiileea** [comment]
[240923.zip](https://github.com/elax46/custom-brand-icons/files/12708090/240923.zip)
latest requests

**Comment by rchiileea** [comment]
[07-10-23.zip](https://github.com/elax46/custom-brand-icons/files/12837847/07-10-23.zip)

**Reply by elax46** [reply]
@rchiileea weather-page-alt have a multiple path

**Reply by rchiileea** [reply]
sorry just seen this I dont understand why I dont get notifications

**Comment by rodolfomachado** [comment]


**Reply by elax46** [reply]
Open a issues

**Comment by rchiileea** [comment]
[24-10-23.zip](https://github.com/elax46/custom-brand-icons/files/13121777/24-10-23.zip)
all latest icons done and corrected weather-page-alt (forgot to remove the bleed border I have on my template for ha compatitbility).

**Reply by elax46** [reply]
`soil-reading` have a multiple path

**Comment by rchiileea** [comment]
[29-10-23.zip](https://github.com/elax46/custom-brand-icons/files/13197636/29-10-23.zip)
fixed and new icons done, changed the request format after someone on discord tried to hack me with a svg link, so now stated no links in request please upload svg and or image (especially as some svg's sent are like original mario bros graphics lol

**Comment by elax46** [comment]
> [29-10-23.zip](https://github.com/elax46/custom-brand-icons/files/13197636/29-10-23.zip) fixed and new icons done, changed the request format after someone on discord tried to hack me with a svg link, so now stated no links in request please upload svg and or image (especially as some svg's sent are like original mario bros graphics lol

`aqara-e1` have `rotate(-21.155)`

**Reply by rchiileea** [reply]
adobe must be playing up when saving, Ill re install it

**Comment by rchiileea** [comment]
Fixed and added some more alex items
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/1605413f-bbf1-44c6-a93f-8ec9e7b1f909)
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/e7fd8fcd-7f5f-4598-bb24-a14c5d3bbfbb)
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/4fb1709c-d117-4a3e-9fa0-a1e6a3d65b5a)
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/5eda140f-d64d-4f00-bd5d-18ed1cf23065)

[30-10-23.zip](https://github.com/elax46/custom-brand-icons/files/13203304/30-10-23.zip)

**Comment by rchiileea** [comment]
[30-11-23.zip](https://github.com/elax46/custom-brand-icons/files/13512150/30-11-23.zip)

**Comment by rchiileea** [comment]
[08-12-23.zip](https://github.com/elax46/custom-brand-icons/files/13614286/08-12-23.zip)

**Comment by rchiileea** [comment]
[080124.zip](https://github.com/elax46/custom-brand-icons/files/13864043/080124.zip)

was ill over christmas (well from 27th onwards had bloody covid.

**Comment by elax46** [comment]
> [080124.zip](https://github.com/elax46/custom-brand-icons/files/13864043/080124.zip)
> 
> was ill over christmas (well from 27th onwards had bloody covid.

`octopusenergy` have a multiple path

**Comment by rchiileea** [comment]
[octopusenergy.zip](https://github.com/elax46/custom-brand-icons/files/13872556/octopusenergy.zip)

**Comment by rchiileea** [comment]
[080124.zip](https://github.com/elax46/custom-brand-icons/files/14083022/080124.zip)
Latest request done

**Comment by rchiileea** [comment]
bump

**Reply by elax46** [reply]
### I haven't read the notifications of the last few hours, thanks for your support as always but they seem to me to be icons already inserted
